protocol/x/prices/keeper/slinky_adapter.go

```diff
@@ -0,0 +1,72 @@
+package keeper
+
```

Comment:
nit: let's move this to a separate dir, /adapters?

Reply:
We can. There's only one there for now--not sure if there will be more ever?